### ISSUE TITLE
Search results text should have role attribute as status

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#9188]
### Description:

role=status
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22.html

![image](https://user-images.githubusercontent.com/20179540/91288866-1f618800-e7af-11ea-988a-149257c0fc3c.png)

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---

**Comment by erangatl** [comment]
Search results count is not shown in the current UI, This is a UX issue
![image](https://user-images.githubusercontent.com/20179540/114298865-03102280-9ad6-11eb-95d2-1be6d268ad15.png)
